xmppserver/src/main/webapp/session-row.jspf

```diff
@@ -58,21 +58,21 @@
     </td>
     <% } %>
     <td>
-        <%  int _status = sess.getStatus();
+        <%  Session.Status _status = sess.getStatus();
             boolean isDetached = sess instanceof LocalSession && ((LocalSession) sess).isDetached();
             if (isDetached) { %>
 
             <fmt:message key="session.details.sm-detached" />
         <%
-            } else if (_status == Session.STATUS_CLOSED) {
+            } else if (_status == Session.Status.CLOSED) {
```

Comment:
`sess.isClosed()` ?

Reply:
Not disagreeing, but: https://github.com/igniterealtime/Openfire/pull/2165#discussion_r1071926857